apps/talk_signaling.sh

```diff
@@ -104,7 +104,7 @@ else
 fi
 
 # Let the user choose port. TURN_PORT in msg_box is taken from lib.sh and later changed if user decides to.
-NONO_PORTS=(22 25 53 80 443 3306 5178 5179 5432 7983 8983 10000 8081)
+NONO_PORTS=(22 25 53 80 443 1024 3012 3306 5178 5179 5432 7983 8983 10000 8081)
```

Comment:
Great catch! Please add that to the original script as well.

@morph027 Can you please list all the ports in use by the Signaling server?

Reply:
they were already added to the original talk script 👍